src/Microsoft.DncEng.Configuration.Extensions/Microsoft.DncEng.Configuration.Extensions.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <IsPackable>true</IsPackable>
     <TargetFrameworks>netcoreapp3.1;netcoreapp2.1</TargetFrameworks>
+    <Nullable>enable</Nullable>
```

Comment:
Why enable?  You keep slipping nullable into weird places...

Reply:
Especially when the target framework is 2.1, this doesn't make a lot of sense.

Reply:
This project already had it enabled..... I am not going to remove them all from this one.